stable_baselines/common/policies.py

```diff
@@ -50,18 +50,23 @@ def mlp_extractor(flat_observations, net_arch, act_fun):
     :param net_arch: ([int or dict]) The specification of the policy and value networks.
         See above for details on its formatting.
     :param act_fun: (tf function) The activation function to use for the networks.
+    :param all_acts: (bool) whether to return lists of activations for each layer, or just the final activations.
```

Comment:
This parameter doesn't seem very useful to me: the user never calls `mlp_extractor` directly, and from inside Baselines I think we always call it with it set to `all_acts=True`?

I suggest we just get rid of it, and always consider the case of `all_acts=True`, which also simplifies this function. Note that this just extracts TensorFlow operations that, if run, will reveal the activations. We do not have to use them!

We can then change `step` in the `Transparent*Policy` classes to let us specify which activations we want to see instead?